event/event.go

```diff
@@ -0,0 +1,238 @@
+    // It varies between 0.0 for not pressed, and 1.0 for completely pressed.
+    Pressure float32
+}
+
+// MouseButtonDown is a mouse button Release event.
+// The data is identical to a MouseButtonDown event.
+type MouseButtonUp MouseButtonDown
+
+// MouseEnter occurs when the mouse enters the view window.
+type MouseEnter struct {
+    // X is the X position of the mouse pointer.
+    // This value is expressed in device independent pixels.
+    X float32
+    // Y is the Y position of the mouse pointer.
+    // This value is expressed in device independent pixels.
+    Y float32
+}
+
+// MouseLeave occurs when the mouse leaves the view window.
+// The data is identical to MouseEnter
+type MouseLeave MouseEnter
+
+// ViewUpdate occurs when the application is ready to update
+// the next frame on the view port.
+type ViewUpdate struct {
+    // No data neccesary, for now.
+}
+
+// ViewSize occurs when the size of the application's view port changes.
+type ViewSize struct {
+    // Width is the width of the view.
+    // This value is expressed in device independent pixels.
+    Width int
+    // Height is the height of the view.
+    // This value is expressed in device independent pixels.
+    Height int
+}
+
+// TouchBegin occurs when a touch begins.
+type TouchBegin struct {
+    // ID identifies the touch that caused the touch event.
+    ID int
+    // X is the X position of the touch.
+    // This value is expressed in device independent pixels.
+    X float32
+    // Y is the Y position of the touch.
+    // This value is expressed in device independent pixels.
+    Y float32
+    // Pressure is the pressure applied to the touch.
+    // It varies between 0.0 for not pressed, and 1.0 for completely pressed.
+    Pressure float32
+    // Primary represents whether the touch event is the primary touch or not.
+    // If it is true, then it is a primary touch.
+    // If it is false then it is a secondary touch. 
+    Primary bool
+}
+
+// TouchMove occurs when a touch moved, or in other words, is dragged.
+type TouchMove struct {
+    // ID identifies the touch that caused the touch event.
+    ID int
+    // X is the X position of the touch.
+    // This value is expressed in device independent pixels.
+    X float32
+    // Y is the Y position of the touch.
+    // This value is expressed in device independent pixels.
+    Y float32
+    // DeltaX is the change in X since last touch event.
+    // This value is expressed in device independent pixels.
+    DeltaX float32
+    // Deltay is the change in Y since last touch event.
+    // This value is expressed in device independent pixels.
+    DeltaY float32
+    // Pressure of applied touch.
+    // It varies between 0.0 for not pressed, and 1.0 for completely pressed.
+    Pressure float32
+    // Primary represents whether the touch event is the primary touch or not.
+    // If it is true, then it is a primary touch.
+    // If it is false then it is a secondary touch.
```

Comment:
`Otherwise, it is false` is more precise (there can be third touch)

Reply:
ok